docs/using-the-jdbc-driver/using-plugins/UsingTheFailover2Plugin.md

```diff
@@ -43,7 +43,7 @@ With the `failover2` plugin:
 - The `MonitoringRdsHostListProvider` continues topology monitoring at an increased rate to ensure all cluster nodes appear in the topology.
 
 ## Using the Failover Plugin v2
-The Failover Plugin v2 will **NOT** be enabled by default. It must be explicitly included by adding the plugin code `failover2` to the [`wrapperPlugins`](../UsingTheJdbcDriver.md#aws-advanced-jdbc-driver-parameters) value, or by adding it to the current [driver profile](../UsingTheJdbcDriver.md#connection-plugin-manager-parameters). After you load the plugin, the failover feature will be enabled. 
+The Failover Plugin v2 will be enabled by default if the [`wrapperPlugins`](../UsingTheJdbcDriver.md#connection-plugin-manager-parameters) value is not specified. The failover plugin v2 can also be explicitly included by adding the plugin code `failover2` to the [`wrapperPlugins`](../UsingTheJdbcDriver.md#aws-advanced-jdbc-driver-parameters) value, or by adding it to the current [driver profile](../UsingTheJdbcDriver.md#connection-plugin-manager-parameters). After you load the plugin, the failover feature will be enabled. 
```

Comment:
> The failover plugin v2 can also be explicitly included...

This phrase confuses me a bit. The previous sentence says it's included by default. And then this sentence says it should be explicitly added.